app/adapters/request-config.js

```diff
@@ -0,0 +1,25 @@
+import AdapterError from '@ember-data/adapter/error';
+import V3Adapter from 'travis/adapters/v3';
+
+export default V3Adapter.extend({
+  queryRecord(store, type, { id, data }) {
+    const url = `${this.buildURL('repo', id)}/request/config`;
+    return this.ajax(url, 'POST', { data });
+  },
+
+  handleResponse(status, headers, body) {
+    if (status == 400) {
+      let error = JSON.parse(body.error_message).error;
```

Comment:
`JSON.parse` may throw error if it can't parse the string, and that would not be handled. Maybe worth wrapping to `try-catch` block?

Reply:
we're controlling this on the api ourselves though ... so i wasn't sure it's needed? i can add it, of course.

Reply:
done: https://github.com/travis-ci/travis-web/commit/44cd613529f3b30ff392d3bf9ce5330f35e9d0b2

Reply:
@svenfuchs thank you! The only note that I have is that you can reuse existing helper: https://github.com/travis-ci/travis-web/blob/master/app/utils/json-parser.js